.github/workflows/container.yml

```diff
@@ -0,0 +1,39 @@
+name: Container Check
+
+on:
+  push:
+    paths-ignore:
+      - '**/*.md'
+  pull_request:
+    paths-ignore:
+      - '**/*.md'
+
+jobs:
+  container-check:
+    runs-on: ubuntu-latest
+    if: (github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository)
+    steps:
+      - name: Clone the code
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '~1.22'
+
+      - name: Prepare project-v4
+        working-directory: testdata/project-v4
+        run: go mod tidy
```

Comment:
The task is to use docker-buildx, check the Docker images, and validate whether the image has been generated for all platforms.

However, I believe this is related to the changes proposed in [this PR](https://github.com/kubernetes-sigs/kubebuilder/pull/4289).

WDYT about we do that in another way? 
What do you think about running the necessary commands in the PR to ensure everything is still working as expected after your changes? Additionally, you could include the results in the PR description to demonstrate that it works as intended.

Reply:
I can do that yes

But would not be nice to have this automated?

If not, all PR that involve container related changes would require one more step, right?